prov/verbs/src/fi_verbs.c

```diff
@@ -844,6 +875,15 @@ fi_ibv_msg_ep_recvv(struct fid_ep *ep, const struct iovec *iov, void **desc,
 	return fi_ibv_msg_ep_recvmsg(ep, &msg, 0);
 }
 
+static inline int
+fi_ibv_post_send(struct ibv_qp *qp, struct ibv_send_wr *wr,
+		struct ibv_send_wr **bad_wr)
+{
+	int ret;
+	ret = ibv_post_send(qp, wr, bad_wr);
+	return (ret == ENOMEM) ? -FI_EAGAIN : -ret;
```

Comment:
I've removed this from the PR and will submit a separate PR for this, so that we can discuss it separately.